REWRITES.md

```diff
@@ -0,0 +1,14 @@
+> This document outlines the changes between the old [Holoitems](https://github.com/dlee13/OldHoloItems) and the rewrite.
+> Sorted Chronologically
+
+# Magnet
+(Please fill in, I haven't been following at this point)
+
+# Tide Rider
+Fixed a bug where players could scale a wall with the Tide Rider.
+
+# Backdash
+Implemented without changes to functionality.
+
+# Memento
+Implemented without changes to functionality.
```

Comment:
We renamed Momento (which isn't a word) to Memento. We now call `PlayerDeathEvent::setShouldDropExperience`. I don't know whether that fixed an exp dupe bug or whether old HoloItems doesn't have a dupe bug in case setting keep level also prevents exp from dropping. Also, the item is now officially stackable.